### DISCUSSION TITLE
Tracing information is getting lost for a few requests during load test

### DISCUSSION THREAD

**Discussion by MALPI** [open-telemetry/opentelemetry-java-instrumentation#6539] [Q&A]
Hey there,

we instrumented a few of our services with the Agent. Recently we were running load tests and experienced some problems.

Let me share a bit of context:

In order to mark the requests as originating from the load test we are making use of baggage items.
These are supposed to be propagated together with the tracing information.

Since the service is instrumented with the agent, we see a low fraction of requests that are loosing the **complete** tracing information. Like all attributes, traceid, spanid, baggage item, and tracing is initiating a new context with a new trace id.

The service is running on a Tomcat with Spring Boot and apache http client. 
I'm unable to reproduce this behavior locally

From the last test that we've run we had overall 269 requests. 252 were properly instrumented, 17 lost the tracing context.

Actually, there is nothing special for those 17 requests. Same implementation same client same load test tool, same endpoint.
I'm absolutely clueless what's going on there and hope that you might be able to give me decisive hint.

**Comment by trask** [comment]
x-posted to https://cloud-native.slack.com/archives/C0150QF88FL/p1662377807209569
